packages/lit-dev-content/site/docs/v3/tools/publishing.md

```diff
@@ -78,29 +78,26 @@ To compile a Lit component that uses proposed JavaScript features not yet includ
 Install Babel and the Babel plugins you need. For example:
 
 ```sh
-npm install --save-dev @babel/core
-npm install --save-dev @babel/plugin-proposal-class-properties
-npm install --save-dev @babel/plugin-proposal-decorators
+npm install --save-dev @babel/core @babel/cli @babel/preset-env @babel/plugin-proposal-decorators
```

Comment:
So there isn't a horizontal scroll bar, is there an alternative where this is broken up over multiple lines, maybe using `\`? Although this is a nit, and the newline variant may be more distracting.

Reply:
Good call! I like the multi-line better.
<img width="811" alt="image" src="https://github.com/lit/lit.dev/assets/40413829/06d815eb-4d90-4f41-adde-ab773d7b482c">